yaml-validation/models/validation.go

```diff
@@ -0,0 +1,49 @@
+package models
+
+import (
+	"fmt"
+	"github.com/gin-gonic/gin"
+	"github.com/instrumenta/kubeval/kubeval"
+	"net/http"
+)
+
+type Content struct  {
+	Content string `form:"content" json:"content"`
+}
+
+func main() {
+
+
+}
+
+func GetValidationResult(c *gin.Context) {
```

Comment:
Move this to its own router file in the router folder. That file should probably be named `validation.go`